universal-application-tool-0.0.1/app/controllers/dev/FileUploadController.java

```diff
@@ -50,7 +50,7 @@ public FileUploadController(
   }
 
   public Result index(Request request) {
-    if (!(isDevEnvironment() || isStaging())) {
+    if (!isDevEnvironment()) {
```

Comment:
question (blocking): do we not want this on staging?

Reply:
ahh yeah so i looked back at the isDevEnvironmnet and it does the isStaging request

Reply:
https://github.com/seattle-uat/civiform/blob/main/universal-application-tool-0.0.1/app/controllers/dev/DevController.java#L29

Reply:
right on